<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOG IN. PLEASE</string>
    </property>
   </widget>
   <widget class="QLabel" name="policemen_picture">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>401</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logos/policeman.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLog_in">
    <property name="title">
     <string>Log in</string>
    </property>
    <addaction name="separator"/>
    <addaction name="log_as_admin"/>
    <addaction name="log_as_ticketsman"/>
    <addaction name="log_as_driver"/>
   </widget>
   <addaction name="menuLog_in"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="log_as_admin">
   <property name="text">
    <string>Log as Admin</string>
   </property>
  </action>
  <action name="log_as_ticketsman">
   <property name="text">
    <string>Log as Ticketsman</string>
   </property>
  </action>
  <action name="log_as_driver">
   <property name="text">
    <string>Log as Driver</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
